src/data_provider/src/sysInfoLinux.cpp

```diff
@@ -355,6 +364,118 @@ nlohmann::json SysInfo::getNetworks() const
     return networks;
 }
 
+std::pair<std::string, std::string> parseProcessInfo(char path[])
```

Comment:
What happens if  any of the following `find()` fails?

Reply:
Added a check for that